system_metrics_collector/src/system_metrics_collector/main.cpp

```diff
@@ -59,29 +53,21 @@ int main(int argc, char ** argv)
   using namespace std::chrono_literals;
   const auto cpu_node = std::make_shared<system_metrics_collector::LinuxCpuMeasurementNode>(
     "linuxCpuCollector",
-    kDefaultCollectPeriod,
-    kStatisticsTopicName,
-    kDefaultPublishPeriod);
+    rclcpp::NodeOptions());
 
   const auto mem_node = std::make_shared<system_metrics_collector::LinuxMemoryMeasurementNode>(
     "linuxMemoryCollector",
-    kDefaultCollectPeriod,
-    kStatisticsTopicName,
-    kDefaultPublishPeriod);
+    rclcpp::NodeOptions());
```

Comment:
I think this is the default behavior, we could just pass the name instead? (ditto for all the other instances).

Reply:
I'll add a default `rclcpp::NodeOptions` parameter in the header file for each type of node.

Reply:
Done.